src/router/utils/HelperFunctions.jsx

```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
+import Toast from "../../components/Toast";
+
+export const addToWatchlist = async (video, videoDispatch) => {
+  try {
+    const response = await axios.post(
+      "/api/user/watchlater",
+      { video },
+      {
+        headers: {
+          authorization: localStorage.getItem("userToken"),
+        },
+      }
+    );
+    if (response.status === 201) {
+      videoDispatch({
+        type: "ADD_TO_WATCHLIST",
```

Comment:
Try not to use hardcoded values for types and it might can issue if any letter is missed or casing is incorrect